platform/ios/src/MGLMapView.mm

```diff
@@ -4249,11 +4249,7 @@ - (void)setSelectedAnnotations:(NS_ARRAY_OF(id <MGLAnnotation>) *)selectedAnnota
 
     if ([firstAnnotation isKindOfClass:[MGLMultiPoint class]]) return;
 
-    // Select the annotation if it’s visible.
-    if (MGLCoordinateInCoordinateBounds(firstAnnotation.coordinate, self.visibleCoordinateBounds))
-    {
-        [self selectAnnotation:firstAnnotation animated:NO];
-    }
+    [self selectAnnotation:firstAnnotation animated:NO];
```

Comment:
Would you mind making the corresponding change to the macOS implementation of MGLMapView? Thanks!

Reply:
Will do.